asteroid/utils/deprecation_utils.py

```diff
@@ -48,3 +49,49 @@ def wrapped(*args, **kwargs):
         return wrapped
 
     return decorator
+
+
+def is_overridden(method_name, obj, parent=None) -> bool:
+    """Check if `method_name` from parent is overridden in `obj`.
+
+    Args:
+        method_name (str): Name of the method.
+        obj: Instance or class that potentially overrode the method.
+        parent: parent class with which to compare. If None, traverse the MRO
+            for the first parent that has the method.
+
+    Raises RuntimeError if `parent` is not a parent class and if `parent`
+    doesn't have the method. Or, if `parent` was None, that none of the
+    potential parents had the method.
+    """
+
+    def get_mro(cls):
+        try:
+            return inspect.getmro(cls)
+        except AttributeError:
+            return inspect.getmro(cls.__class__)
+
+    def first_parent_with_method(fn, mro_list):
+        for cls in mro_list[::-1]:
+            if hasattr(cls, fn):
+                return cls
+        return None
+
+    if not hasattr(obj, method_name):
+        return False
+
+    instance_attr = getattr(obj, method_name)
+    if not instance_attr:
```

Comment:
```suggestion
    try:
        instance_attr = getattr(obj, method_name)
    except AttributeError:
        return False
```

Reply:
Otherwise, 1) will raise `AttributeError` if the attr doesn't exist, and 2) the check will consider falsey values as missing.